content/en/docs/device-provisioning-service/features.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Feature list'
+description: 'What features provides Device Provisioning Service?'
+date: '2023-05-19'
+categories: [zero-touch, provisioning]
+keywords: [provisioning, TPM, manufacturer-certificate]
+weight: 2
+---
+
+### Implemented features
+
+- **&#x2611; Device attestation via certificates:** This feature allows devices to prove their identity using [certificates](../attestation-mechanisms.md#x509-certificates).
+- **&#x2611; Onboarding devices to plgd hub:** This feature enables the process of adding devices to the for management and control by [plgd hub](../../architecture/component-overview.md).
```

Comment:
This feature enables the process of adding devices <s>to the</s> for management and control by

Reply:
fixed